### ISSUE TITLE
Format Message Types for EKF

### ISSUE THREAD

**Issue by libBAS** [GOFIRST-Robotics/NASA-RMC-2020-NorthstarRobotics#13]
The ROS [robot_localization](http://wiki.ros.org/robot_localization) package uses an EKF to fuse sensor data.  The [message type for data coming from each sensor](http://docs.ros.org/melodic/api/robot_localization/html/preparing_sensor_data.html) should be as follows:

- IMU - [sensor_msgs/Imu](http://docs.ros.org/api/sensor_msgs/html/msg/Imu.html)

- Decawave - [geometry_msgs/PoseWithCovarianceStamped](http://docs.ros.org/api/geometry_msgs/html/msg/PoseWithCovarianceStamped.html)

- Wheel Odometry - [nav_msgs/Odometry](http://docs.ros.org/api/nav_msgs/html/msg/Odometry.html)

**Comment by libBAS** [comment]
Additionally, aruco localization

**Comment by JWCS** [comment]
Remember there's two EKF's; 
What topics do you want to have published / named for both of them? 
Y'all should start a launch/config file pair for both of them to target

**Comment by juliaschatz** [comment]
Right now I have Aruco producing a PoseStamped where the pose is the robot pose. I can probably make this a PoseStampedWithCovariance with a little work if that's necassary.

**Comment by JWCS** [comment]
Yeah, the covariance, or at least a variance diagonal, is critical.

**Comment by juliaschatz** [comment]
Okay, now we have PoseWithCovarianceStamped messages.

**Comment by libBAS** [comment]
Current status:

- Lower level node - Done (runs, launches, tested)

- Upper level node - Math is written, currently being put into a node

Design of upper node - update position from each data point, taking into account current state
